101-vm-with-data-management-gateway/azuredeploy.json

```diff
@@ -0,0 +1,297 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "existingDataFactoryName": {
+      "type":"string",
+      "metadata": {
+        "description": "your existing data factory name"
+      }
+    },
+    "GatewayName": {
```

Comment:
Now gateway is a sub resource in data factory, but we prepare to make it as ARM top-level resource in the following months, so the gateway name should be unique in subscription.  Sorry, I am not sure why creating new resource doesn't match the description?